modules/docs/arrow-docs/docs/docs/arrow-fx/integrations/kotlinxcoroutines/README.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: docs-core
+title: kotlinx.coroutines
+permalink: /docs/integrations/kotlinxcoroutines/
+---
+
+## Kotlin Coroutines vs runtime
+
+Due to the current nature of Kotlin Coroutines' implementation we need to make a differentiation between the Coroutines feature in the language and the runtime, in order to understand their differences and this module altogether:
+
+* Suspend functions and implicit continuations are built in the language, meaning that you don't need extra dependencies to use those.
+* Coroutines runtime and cancellation are built as a separate module: KotlinX Coroutines. Which is used to actually execute your Coroutines.
+
+Due to this differentiation there are 2 possible integration options: to use suspend functions with Arrow Fx or use Arrow (specifically IO) with the Coroutines runtime. 
```

Comment:
Link to IO here maybe

Reply:
I don't mind linking it, although I'd like to note that since we're in the Fx module itself, none of the other docs seem to link to IO as it's "implicit" to the module itself I believe